tests/loader_regression_tests.cpp

```diff
@@ -3208,31 +3208,31 @@ TEST(SortedPhysicalDevices, DevicesSortedDisabled) {
     ASSERT_EQ(VK_SUCCESS, instance->vkEnumeratePhysicalDevices(instance, &device_count, physical_devices.data()));
     ASSERT_EQ(device_count, max_phys_devs);
 
-    // make sure the order is what we started with - but its a bit wonky due to the loader reading physical devices "backwards"
```

Comment:
Do these test changes, loader changes, and the prior test fixes actually stand alone? It seems like if this isn't squashed, tests are going to fail in between these commits.